src/api/whitelist.js

```diff
@@ -38,6 +38,22 @@ export default function() {
     });
 
   });
+  
+ //get white plate by vehicle number
+  api.get('/number/:number', isAuthenticated, (req, res) => {
```

Comment:
kaip ir sakiau, pervadinčiau šitą į kokį nors `/api/whitelist/search?plate=GOV123`
parametrą `plate` gali nusiskaityti iš `req.query.plate`